evalml/automl/automl_search.py

```diff
@@ -771,7 +789,9 @@ def _handle_keyboard_interrupt(self):
                 .lower()
             )
             if choice == "y":
-                logger.info("Exiting AutoMLSearch.")
+                self._activate_logger
+                self.logger.info("Exiting AutoMLSearch.")
+                self._deactivate_logger
```

Comment:
Warning: missing parentheses to invoke the methods here

I'm surprised our lint didn't catch this! I wonder if there's a way / file an issue to have lint fail if we try to invoke a non-property as a property. Tagging @angela97lin since she's been messing with linting / style recently

Reply:
That's a really good question @dsherry! I'm not sure I've run into anything that would have caught this--probably tricky to do since calling this without the paraentheses still returns the method and is "valid" 🤔 . Will keep an eye out though!